### DISCUSSION TITLE
Linux command line interface usage

### DISCUSSION THREAD

**Discussion by jaychoubaby** [spacedriveapp/spacedrive#1522] [Feature Request]
Spacedrive amazed me. When I tried to install it using Linux, I found that Linux seems to require a graphical interface in order to use it. I think it should be able to support a non-graphical interface for usage. I hope it can be supported. Thank you very much.

**Comment by HeavenVolkoff** [comment]
A CLI interface is planned, but low priority for now. There is a not much tested, very bare-bones, implementation here: https://github.com/spacedriveapp/spacedrive/tree/main/apps/cli. PRs for this are welcomed
